client/components/ReviewRating/WriteReview/WriteReview.jsx

```diff
@@ -49,132 +60,215 @@ const WriteReview = ({
-          <input className="form_input" name="summary" id="summary" value={newReview.summary || ''} onChange={onChangeHandler} maxLength="60" required />
-        </div>
-        <div className="input_label">
-          <span>
-            Body:
-          </span>
-          <textarea className="form_input" name="body" id="body" value={newReview.body || ''} onChange={onChangeHandler} maxLength="2500" row="10" required />
-        </div>
-        <div className="input_label">
-          Recommend:
-          <span htmlFor="yes">
-            Yes
-            <input name="recommend" id="yes" type="radio" value={true || ''} onChange={onChangeHandler} checked />
-          </span>
-          <span htmlFor="no">
-            No
-            <input name="recommend" id="no" type="radio" value={false || ''} onChange={onChangeHandler} />
-          </span>
-        </div>
-        <div className="input_label">
-          <span>
-            Name:
-          </span>
-          <input className="form_input" name="name" id="name" value={newReview.name || ''} onChange={onChangeHandler} />
-        </div>
-        <div className="input_label">
-          <span>
-            Email:
-          </span>
-          <input className="form_input" name="email" id="email" value={newReview.email || ''} onChange={onChangeHandler} required />
-        </div>
-        <div className="input_label">
-          <span>
-            Photos:
-          </span>
-          <input name="photos" id="photos" value={newReview.photos || ''} onChange={onChangeHandler} />
-          <button type="button" onClick={saveUrl}>add url</button>
-        </div>
-        {photoUrl && photoUrl.map((url) => (
-          <div key={url}>{url}</div>
-        ))}
-        <label
-          htmlFor="characteristics"
-          className="characteristics_label"
-          id="url_box"
-        >
-          {charName && charName.map((char, index) => (
-            <div key={char}>
-              <label
-                htmlFor={char}
-              >
-                {char}
-              </label>
-              <input className="form_input" value={charValue[index] || ''} name={char} id={char} type="range" min="1" max="5" onChange={onChangeHandler} />
-            </div>
+      {!isValid && (
+        <div id="error_message">
+          <div>You must enter the following:</div>
+          {errMsg[0] && errMsg[0].map((err) => (
+            <div key={err}>{err}</div>
           ))}
-        </label>
-        <div className="review_btn">
-          <button type="button" onClick={() => closeWriteReview()}>Cancel</button>
-          <button type="button" onClick={onSubmit}>Submit</button>
+          <button
+            type="button"
+            onClick={() => {
+              setErrMsg([]);
+              setIsValid(true);
+            }}
+          >
+            OK
+          </button>
         </div>
+      )}
+      <div className={isValid ? 'window' : 'close'}>
+
+        <form className="modal">
```

Comment:
Since our app has more than one modal, it might be best to use a more descriptive class name.